cli/azd/cmd/testdata/TestUsage-azd-template-source-add.snap

```diff
@@ -0,0 +1,21 @@
+
+Adds an azd template source at the specified key (Beta)
+
+Usage
+  azd template source add <key> [flags]
+
+Flags
+        --docs            	: Opens the documentation for azd template source add in your web browser.
+    -h, --help            	: Gets help for add.
+    -l, --location string 	: Location of the template source
+    -n, --name string     	: Name of the template source
+    -t, --type string     	: Kind of the template source
```

Comment:
Would I need to specify `-t url` in the above example? Or is that the default? 

We may want to add some examples below to help guide folks? Or add more details about what accepted values look like?

Reply:
The most common types would be `url` and `file`.  We could look at the location value and attempt to infer the type to also make it a bit easier.

But right now it would be something like `azd template source add foo -t url -l http://example.com/templtes.json`